### PR TITLE
Additional Glass Doors + Secure Windoors

### PR DESCRIPTION
**Pull request by musicmanvr**
Because we all need a bit more door in life.

## About the PR

Service, Virology and Bar now have secure windoors as an option. Janitor and theatre have glass airlock doors now as well.

Atmos door in the airlocks file has been moved to be alphabetical, it's not removed.
## Why / Balance

Useful for mapmaking and allows more options for mapmakers sectioning areas off by access.
## Technical details

Only YAML.
## Media

The doors look identical to existing doors minus their name and access level.

- [  X  ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by DangerRevolution** [comment]
Very nice, very useful for mapping o7

File: Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml [hunk 1]
```diff
@@ -233,8 +257,8 @@
 
 - type: entity
   parent: WindoorSecure
-  id: WindoorSecureAtmosphericsLocked
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml:236]: Why are we removing Atmos door?

**Comment by UbaserB** [comment]
I need a detective access windoor while you’re at it

File: Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml [hunk 2]
```diff
@@ -233,8 +265,8 @@
...
-  suffix: Atmospherics, Locked
+  id: WindoorSecureVirologyLocked
+  suffix: Virology, Locked
```
Comment (Nairodian) [inline Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml:269]: Virology access doesn't exist currently, and may never will. So it's probably best just to not include this and leave this for if/when it gets added.

Comment (musicmanvr) [reply Resources/Prototypes/Entities/Structures/Doors/Windoors/windoor.yml:269]: fair, i'll remove em'

**Comment by Emisse** [comment]
doesnt need a changelog, non playerfacing

**Comment by musicmanvr** [comment]
> doesnt need a changelog, non playerfacing

edited, :)

**Pull request merged**